packages/chain-events/services/ChainEventsConsumer/ChainEventHandlers/storage.ts

```diff
@@ -58,7 +61,19 @@ export default class extends IEventHandler {
   }
 
   private async _shouldSkip(event: CWEvent): Promise<boolean> {
-    return !!this._filterConfig.excludedEvents?.includes(event.data.kind);
+    if (!event) return true;
+    const entity = eventToEntity(event.network, event.data?.kind);
+    if (!entity) return true;
+
+    // filter out all events that won't have an associated entity in the db
+    const [_, kind] = entity
+    if (![EntityEventKind.Create,
```

Comment:
I believe we can simply check if "kind" is null or not here, as "kind" only exists on entity-related events.

Reply:
adding change is easy, lets discuss once on Monday though about this:
> IF we want to continue emitting notifications, which is my intuition, then we will need to refactor the NotificationHandler to no longer depend on having a dbEvent available.

Reply:
There is no need for this check at all given that if `entity` is `null` then we know the event is not related to any entity. If `entity` is defined then we know we don't want to skip the event hence making this part entirely redundant.

Reply:
@jnaviask I pushed a change that cleans this up.